back/engines/commercial/user_custom_fields/spec/acceptance/stats_users_spec.rb

```diff
@@ -498,6 +502,81 @@ def xlsx_worksheet_to_array(worksheet)
       end
     end
 
+    get 'web_api/v1/stats/users_by_custom_field/:custom_field_id' do
+      time_boundary_parameters self
+      group_filter_parameter self
+      participant_filter_parameter self
+      parameter :project, 'Project ID. Only return users that have participated in the given project.', required: false
+      describe 'with filter by participation' do
+        before do
```

Comment:
You can also put this `before` block higher before `describe` to avoid duplication and increase readability if you want.

Reply:
right, they are exactly the same, thanks